src/itest/resources/features/OfficerDelta.feature

```diff
@@ -2,20 +2,25 @@ Feature: Officer Delta
 
 Scenario: Can transform and send a natural officer
   Given the application is running
-  When the consumer receives a natural officer delta
+  When the consumer receives a natural officer delta with id EcEKO1YhIKexb0cSDZsn_OHsFw4
   Then a PUT request is sent to the appointments api with the transformed data
 
 Scenario: Can transform and send a corporate officer
   Given the application is running
-  When the consumer receives a corporate officer delta
+  When the consumer receives a corporate officer delta with id EcEKO1YhIKexb0cSDZsn_OHsFw4
   Then a PUT request is sent to the appointments api with the transformed data
 
 Scenario: Can transform and send a pre-1992-corporate officer
   Given the application is running
-  When the consumer receives a pre_1992_corporate officer delta
+  When the consumer receives a pre_1992_corporate officer delta with id EcEKO1YhIKexb0cSDZsn_OHsFw4
   Then a PUT request is sent to the appointments api with the transformed data
 
 Scenario: Can transform and send a pre-1992-natural officer
   Given the application is running
-  When the consumer receives a pre_1992_natural officer delta
+  When the consumer receives a pre_1992_natural officer delta with id EcEKO1YhIKexb0cSDZsn_OHsFw4
+  Then a PUT request is sent to the appointments api with the transformed data
+
+Scenario: Can transform and send a corporate-non-corp-kind officer
```

Comment:
What do you mean by corporate-non-corp-kind, both a corporate and not a corporate?